src/qibolab/couplers.py

```diff
@@ -0,0 +1,66 @@
+from dataclasses import dataclass, field
+from typing import List, Optional, Tuple, Union
+
+from qibolab.channels import Channel
+from qibolab.native import TwoQubitNatives
+from qibolab.qubits import Qubit, QubitId
+
+CouplerId = Union[str, int]
+"""Type for Coupler names."""
+
+
+@dataclass
+class Coupler:
```

Comment:
Following the comment above about the pulses, we could consider having a more primitive object/type (potentially empty) that both `Qubit` and `Coupler` inherit from. Pulses could then accept this type so that they work with both `Qubit`s and `Coupler`s.

Reply:
Yeah, that could be a good solution.